### PR TITLE
global spuddy

### PR DESCRIPTION
**Pull request by mliu2003**
implemented global spuddy

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
 const steps = d3.selectAll(".step");
 const svg = d3.select("svg");
+const graphic = d3.select(".graphic");
 const width = svg.node().clientWidth;
 const height = svg.node().clientHeight;
 
+const spuddyText = [
+  "Hi there! My name is Spuddy! I'll be your personal potato buddy today! Try scrolling to see some visualizations, and be sure to click on me for more info!",
```
Comment (potofuu) [inline script.js:8]: image
Is a bit too long on my screen

**Review by potofuu — COMMENTED** [review]
Maybe shrink spuddy a bit, otherwise looks good

**Comment by dxie46** [comment]
I think you forgot to delete the old spuddy from the sprout step

Also if you scroll from potato types back up to greenhouse emissions, the sprout graph is still there

otherwise LGTM

**Review by potofuu — APPROVED** [review]
Looks good

**Review by dxie46 — APPROVED** [review]

**Pull request merged**